models/stg_linkedin_ads__creative_history.sql

```diff
@@ -18,13 +18,14 @@ with base as (
 
     select
         id as creative_id,
-        cast(last_modified_time as {{ dbt_utils.type_timestamp() }}) as last_modified_at,
-        cast(created_time as {{ dbt_utils.type_timestamp() }}) as created_at,
         campaign_id,
-        type as creative_type,
+        type,
         cast(version_tag as numeric) as version_tag,
-        status as creative_status,
-        click_uri
+        status,
```

Comment:
Was it intentional to keep the original name of the `status` field? I noticed we still document in the yml as `creative_status`, but only leverage `status` here. Can we update to be consistent.

Reply:
ah yes intentional -- i will adjust the yml